### PR TITLE
Add sync tests for invalid Debian fixtures

### PR DESCRIPTION
**Pull request by quba42**
Depends on https://github.com/pulp/pulp-fixtures/pull/147

ref #6052
https://pulp.plan.io/issues/6052

Has been fully tested with a local build of the pulp-fixtures container.

### CODE REVIEW

File: pulp_deb/tests/functional/api/test_sync.py [hunk 1]
```diff
@@ -154,29 +167,53 @@ def test_invalid_distribution(self):
...
+        """Sync a repository missing a set of Package indecies.
+
+        All the files associated with the relavant content unit are missing.
+        Assert that the relevant exception is thrown.
+        """
+        with self.assertRaises(PulpTaskError) as exc:
+            self.do_test(url=DEB_INVALID_FIXTURE_URL, architectures="ppc64")
+        error = exc.exception.task.error
+        self.assertIn("No suitable Package index file", error["description"])
+        self.assertIn("ppc64", error["description"])
```
Comment (mdellweg) [inline pulp_deb/tests/functional/api/test_sync.py]: 'indices'?

File: pulp_deb/tests/functional/api/test_sync.py [hunk 2]
```diff
@@ -154,29 +167,53 @@ def test_invalid_distribution(self):
...
+        The Release file next to the missing Package indecies is retained.
+        Assert that the relevant exception is thrown.
         """
         with self.assertRaises(PulpTaskError) as exc:
-            self.do_test(url=DEB_INVALID_FIXTURE_URL)
+            self.do_test(url=DEB_INVALID_FIXTURE_URL, architectures="armeb")
         error = exc.exception.task.error
-        for key in ("mismached", "empty"):
-            self.assertIn(key, error["description"])
+        self.assertIn("No suitable Package index file", error["description"])
```
Comment (mdellweg) [inline pulp_deb/tests/functional/api/test_sync.py]: 'signature'?

File: pulp_deb/tests/functional/api/test_sync.py (hunk 1 above)
Comment (quba42) [reply pulp_deb/tests/functional/api/test_sync.py]: Fixed.

File: pulp_deb/tests/functional/api/test_sync.py (hunk 2 above)
Comment (quba42) [reply pulp_deb/tests/functional/api/test_sync.py]: Fixed. What can I say? I had a no spell day.

**Pull request merged**